collections/ansible_collections/cloudkit/service/roles/manage_agents/tasks/import_agents.yaml

```diff
@@ -0,0 +1,135 @@
+- name: Display node names that will be added as agents
+  ansible.builtin.debug:
+    var: manage_agents_node_names
+
+- name: Get list of node info
+  ansible.builtin.include_role:
+    name: massopencloud.esi.node
+    tasks_from: list_nodes
+  vars:
+    node_names: "{{ manage_agents_node_names }}"
+# registers: node_info_list
```

Comment:
Is `node_info_list` used anywhere other than the task below? If not, I kind of think that it might belong within the `get_non_agent_nodes` task below.